src/main/java/com/nike/cerberus/server/config/CmsConfig.java

```diff
@@ -109,6 +102,7 @@ protected CmsConfig(Config appConfig, PropertiesRegistrationGuiceModule properti
                 new BackstopperRiposteConfigGuiceModule(),
                 new CmsFlywayModule(),
                 new OneLoginGuiceModule(),
+                new AwsStsGuiceModule(),
```

Comment:
Why not put this in the CMS Guice Module?  

Most of these others are separated for a reason but this just seems like other code that is in the CMS Guice Module

Reply:
I looked at the module after writing the above comment.  I don't feel that strongly either way.